### DISCUSSION TITLE
Болотный труп

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#487] [Ideas]
Удалось восстановить вот такого монстра:

![z](https://user-images.githubusercontent.com/39790055/74974735-d7942980-5436-11ea-82b2-25b3cc3af3ea.jpg)

Прописан в скриптах аддона, в файлах игры есть отдельная модель (аналогичная зомби), набор анимаций (косячной) и текстура тела (была ранее задействована мной для штанов рабов). Текстура головы взята из Сиквела.

Можно подумать, где его использовать и в каком количестве. Например, в Яркендаре в четвертой главе. Также следует продумать его характеристики. Он может быть как усиленной версией обычного зомби, так и ослабленной.

**Comment by Kor-Angar** [comment]
Это болотный зомби?

**Reply by Kor-Angar** [reply]
> По сути да

Тогда можно заменить часть зомби в Ярике в 4 главе. Так же можно парочку добавить в Минентале в 5 главе на болото там гдракон был.

**Comment by dosinabox** [comment]
> наверное параметры ему надо поправить.. и доспех бы одеть.
> ![image](https://user-images.githubusercontent.com/49514764/75639771-74c73d00-5c43-11ea-9fb9-5e93c6bb7695.png)
> ![image](https://user-images.githubusercontent.com/49514764/75639778-7b55b480-5c43-11ea-80b7-c30d815b549e.png)
> ![image](https://user-images.githubusercontent.com/49514764/75639782-7db80e80-5c43-11ea-806d-e4eac4332080.png)

**Comment by dosinabox** [comment]
Доспех не нужно - он полностью закроет текстуру тела. А ради нее всего моба и чинили)
Наверное, нужно поднять ему защиту хотя бы до 100/100/50/50. Получится самый сильный вид зомби в игре)
Но главный вопрос - куда их (или его) вставить.

**Comment by dosinabox** [comment]
> Но главный вопрос - куда их (или его) вставить.

Ещё можно на болото лагеря сектантов в расширенной рудниковой долине. Помимо болота Яркендара.

**Comment by Kor-Angar** [comment]
Не.. Не самый сильный. Зомби Равена и зомби Пала помоему сильнее 😀Их можно вставить или вместо кого то. и/или дополнительно.Если вставлять дополнительно, то лучше выставить им маленький уровень, что бы не было дисбаланса.Дополнительно можно вставить на болото в Минентале, там где был болотный дракон. Штук 10 возле сокровещницы. Если вместо то можно вставить на болото в Ярике.

**Comment by Kor-Angar** [comment]
А я так в долину и не ходил. Там у тебя дают гору опыта героя и получился дисбаланс :-( народ на WoP какие не реальные  статы стал прокачивать я сначало не понял как это возможно, а потом понял что они просто там опыт сняли 🙃

**Comment by dosinabox** [comment]
> А я так в долину и не ходил. Там у тебя дают гору опыта героя и получился дисбаланс :-(

Предложения?
С одной стороны мне не жалко. А с другой жалко)

**Comment by Kor-Angar** [comment]
Там только один вариант: опыт не давать за всех новых.

**Comment by Kor-Angar** [comment]
Дело не в том что жалко или не жалко. Дело в том что получается Дисбаланс. Если их оставлять то надо тогда проводить расчёты сколько там опыта во 2,3, 4 глава. Рассчитывать это, усреднять 😀Авторы игры достаточно ловко как то рассчитали опыт. Конечно не идеально, есть свои агрехи, но в целом баланс проработал серъезно. А многие правки к сожалению делаются не с точки зрения математики и расчётов, а как бы интуитивно на глазок. И часто интуитивно на глазок оказывается мимо.

**Comment by dosinabox** [comment]
insert swampzombie

**Comment by Kor-Angar** [comment]
так что решила? итих болотных зомби можно использовать за место тех что сейчас появляются в 4 главе в ярике.

**Reply by dosinabox** [reply]
нужно определиться с характеристиками и тогда можно будет добавлять

**Reply by Kor-Angar** [reply]
можно поставить такие же хар-ки как и у оригинального зомби в доспехах. что бы дисбаланса не было.

**Reply by Kor-Angar** [reply]
> можно поставить такие же хар-ки как и у оригинального зомби в доспехах. что бы дисбаланса не было.

![image](https://user-images.githubusercontent.com/49514764/210561960-a54a4632-2d76-49f7-9832-5442244d4bfd.png)

**Reply by Kor-Angar** [reply]
надо проверить какая у них защита в скриптах, так как у них может плюсоваться защита прототипа + доспеха, а может не плюсоваться :-)

**Reply by Kor-Angar** [reply]
мне вообще тема с суммированием брони НПС и доспеха не нравится. так например из-за этих сумм у зомби из 2-3 главы в доспехах  Торуса общий показатель защиты 120, в то время как у зомби из болот в доспехах Равена 100. так как копию доспехов равена под зомби сделали, а копию доспехов торуса нет :-)

**Reply by Kor-Angar** [reply]
то есть тогда проще сделать для зомби броню с защитой 0, а общую защиту прописать в самой инстанции зомби. так будет по крайней мере понятнее кто за кем идёт и кто сильнее, а кто слабее.

**Comment by Kor-Angar** [comment]
с зомби-паладина можно попробовать снять шлем. может будет забавно.

**Comment by dosinabox** [comment]
Места им так и не нашлось, поэтому временно будут вырезаны.
В скриптах будут закомментированы, а контент оставлю здесь:
[swampzombie.zip](https://github.com/user-attachments/files/15934640/swampzombie.zip)
